### PR TITLE
Diverse renamings prior to #16

### PR DESCRIPTION
**Pull request by marcgs**
Diverse renamings prior to implementation of agera-edc/DataDashboardFork#10:
- Menu items naming consolidation and reordering
- Component names for contract definition page renaming

Example instance: http://mvd461-company2-mvd.northeurope.azurecontainer.io

### CODE REVIEW

File: src/modules/app/app-routing.module.ts [hunk 1]
```diff
@@ -22,22 +22,22 @@ export const routes: Routes = [
...
   {
     path: 'transfer-history-viewer',
     component: TransferHistoryViewerComponent,
     data: {title: 'Transfer History', icon: 'assignment'}
   },
   {
-    path: 'contractdefinition-editor',
-    component: ContractDefinitionEditorComponent,
-    data: {title: 'Contract Definition Editor', icon: 'rule'}
+    path: 'contractdefinition-viewer',
```
Comment (chrislomonico) [inline src/modules/app/app-routing.module.ts:33]: should this be update to `contract-definition-viewer`?

**Review by chrislomonico — CHANGES_REQUESTED** [review]
one naming change seems off; granted this is only from reading through it.  I will run it locally to confirm

**Review by cpeeyush — APPROVED** [review]

**Comment by paullatzelsperger** [comment]
one minor thing, ready to merge once that's resolved.

**Comment by marcgs** [comment]
Merging as @chrislomonico comment has been addressed

**Pull request merged**